internal/init/prompts.go

```diff
@@ -29,6 +30,7 @@ type PromptHandler struct {
 	moduleconfig.Parameter
 	Condition CustomConditionSignature
 	Validate  func(string) error
+	Info      string
```

Comment:
Maybe this should be part of `moduleconfig.Parameter` instead so that modules can provide more context for individual parameters?

Reply:
right that make sense 👍 , didnt think of that